lib/postmark/api_client.rb

```diff
@@ -168,12 +168,18 @@ def delete_trigger(type, id)
     end
 
     def get_triggers(type, options = {})
-      _, batch = load_batch("triggers/#{type}", 'Tags', options)
+      type = type.to_s.strip.downcase.delete('_')
```

Comment:
Sorry, missed this on the first pass. Looks like you could make things simpler by using `Postmark::Inflector.to_postmark` here:

```
type = Inflector.to_postmark(type)
find_each("triggers/#{type.downcase}", type, options)
```

Same can be applied to the triggers method below.

Reply:
Hey! yeah, I was thinking about that, and it would remove the ugly looking comparison I do for string. The reasonging for not doing that was this:

if user doesn't pass a symbol, like: :inbound_rules , in that case this method would not work for user, since if he for example does the call with get_triggers('inboundrules') , it will return empty result.

Though, I guess that is fine? We should force them to use call properly like: get_triggers(:inbound_rules)

Let me know what you think?

Reply:
I think it’s fine. We should just document what available triggers are and how they should be spelled [in the Wiki](https://github.com/wildbit/postmark-gem/wiki/The-Triggers-API-Support) :)

Reply:
agreed, updating :)

Reply:
Made this change. Realized that I actually did not propagate the update to all triggers endpoints, so I added create/list/delete updates for inbound rules and updated tests for it.

I added separate section to triggers page about [inbound rules](https://github.com/wildbit/postmark-gem/wiki/The-Triggers-API-Support).

Let me know if we should update anything else.
I slightly changed that api_client_spec file format, ran Rubymine autoformat - did not change much out of habit